extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -257,6 +262,42 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 
   /**
+   * If requestsFrameUrl (and optionally dataHash as well) are specified in
+   * config, check whether third-party iframe already exists, and if not,
+   * create it.
+   * @private
+   */
+  handleRequestsFrameUrl_() {
+    if (!this.config_['requestsFrameUrl']) {
```

Comment:
I thought we decided this would be specified as part of the transport object?  Also I believe this.config_ at this point is a merging of within GH repo values, potential remote config, and within amp-analytics element config.  How do you plan to ensure that we ONLY allow values from GH configs?